upstream-community-operators/pystol/0.8.17/pystol.v0.8.17.clusterserviceversion.yaml

```diff
@@ -0,0 +1,311 @@
+            "action_stderr": "{}",
+            "action_stdout": "{}"
+          }
+        }
+      ]
+spec:
+  displayName: Pystol
+  description: |
+    **The open source, self-hosted and cloud-native fault injection platform**
+
+    ## Introduction
+
+    Pystol is a fault injection platform represented as a Software Product Line to analyze the effects caused by a wide spectrum of adverse conditions.
+    In particular, this software platform is designed for running on top of cloud-native environments without the distinction of its usage in private or public clouds.
+    The tool is modular enough to be extended as an Ansible collection enabling users to create their own actions without any dependency with the framework.
+
+    Including fault injection actions in the production infrastructure allows companies to have consistent environments, improve applications dependability against
+    unexpected failures, provide better user experience, and an overall improvement of the system's quality.
+    Using pystol, we enable cloud-computing technologies together with development teams to rapidly create complex systems and deploy them continuously, at global scale.
+
+    ## Documentation
+
+    Please refer to the [official documentation](https://docs.pystol.org)
+    website for any information related to the project.
+
+    ## CI dashboard
+
+    Pystol uses **GitHub actions**
+    and **badges** to run all the CI
+    tasks, the result of running these
+    tasks is represented using badges.
+
+    In particular we embrace the use of
+    CI dashboard as information radiators.
+
+    We created the [badgeboad project](https://badgeboard.pystol.org)
+    to show the value of any set of badges as a dashboard.
+
+    For more information you can open the
+    [CI dashboard](https://badgeboard.pystol.org)
+    directly or go to the
+    [project page in GitHub](https://github.com/pystol/badgeboard).
+
+    ## Container images
+
+    All pystol official container images are hosted in Quay.io under
+    the [Pystol organization](https://quay.io/organization/pystol).
+
+    There you will find two repositories:
+
+    * The Pystol [staging repository](https://quay.io/repository/pystol/pystol-operator-staging).
+    Here you will find all the container images from the upstream end-to-end jobs from the GitHub
+    Actions jobs.
+
+    * The Pystol [stable repository](https://quay.io/repository/pystol/pystol-operator-stable).
+    Here you will find all the container images from the stable branches.
+
+    ## License
+
+    Pystol is open source software
+    licensed under the [Apache license](LICENSE).
+
+    ## Requirements
+
+    * When installing Pystol, create and use the **pystol** namespace.
+
+    ## Accessing the native web user interface.
+    * If the user wants to access the native Pystol's UI, create a route in the cluster pointing to the Pystol's service.
+
+  maturity: alpha
+  version: 0.8.17
+  replaces: ''
```

Comment:
Please remove replaces field. Use it only when you upgrade version. Thanks

Reply:
Done